### ISSUE TITLE
Pagination bar is scrolling in mobile view 🐞 Bug

### ISSUE THREAD

**Issue by sa-paul** [CircuitVerse/CircuitVerse#4375]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

https://github.com/CircuitVerse/CircuitVerse/assets/99007201/6dd494ef-2fd9-48b2-9323-894b4480662a

Pagination bar is scrolling in mobile view

### Steps To Reproduce

1. Go to https://circuitverse.org/ in android, or, open in mobile view 
2. Scroll down to the end of the home page
3. See the scrolling pagination bar

### Expected Behavior

Pagination bar should shrink with shrinking screen size.
The numbers shown in pagination bar should also accordingly.  

### Screenshots

https://github.com/CircuitVerse/CircuitVerse/assets/99007201/ef627563-c4dc-4d1e-83bf-d761260b64f1

### Device Information [optional]

```markdown
- OS: Android 11
- Browser: Chrome
- version: 120.0.6099.44
```

### Are you working on this issue?

Yes

**Comment by yogeshsshekhawat** [comment]
@sa-paul bro but it can be a feature too
because it is much easy to go 5 or 6  page next by just scrolling
or it can be a feature not a bug

**Comment by akankshat05** [comment]
can I work on this issue?

**Comment by tanmoysrt** [comment]
Issue verified.
For mobile devices, we can reduce the pages button.

**Comment by sa-paul** [comment]
@syntax-bot Yes, I am continuously working on it and it has also come to an end. Just merging it out.
